### PR TITLE
Modernize text in policy.html and website_updating.html 

### PR DESCRIPTION
**Pull request by libbooze**
Few minor cleanups, nothing major.

 I did not want to get into the trouble for recommending specific browsers so I just suggested 2 modern browsers.

### CODE REVIEW

**Comment by mclow** [comment]
This change looks fine to me; but it should probably go into `master`.

Also please note that we're currently (8/2021) having trouble getting the website to auto-update. I expect this to be resolved shortly (as in in the next week or so).

**Comment by libbooze** [comment]
> 
> 
> This change looks fine to me; but it should probably go into `master`.

Hi Marshall does that mean that website_updating.html is wrong, or I misunderstood it?
_In progress work is on the beta branch, the live website is on the master branch. The website is updated automatically from the git repo. If you want to update the website, please create a pull request._

image

I will try to rebase, not sure if that will works since it seems there is 500 commits difference...

File: development/website_updating.html [hunk 1]
```diff
@@ -98,12 +98,12 @@ <h2><a name="ExistingPage" id="ExistingPage"></a>Existing
...
               test you changes in as many as possible. Preferably at least
-              with IE and one other modern browser if you are on Windows. But
-              if you are on another platform testing with the "default"
-              browser is best. Normally viewing the page in outside of the a
-              web server context will only show you the content and part of
-              the footer. This is because the site uses Server Side Includes
-              (SSI) which most browsers will not process. To look at the
+              two modern browsers if you are on Windows. But if you are on another
+              platform testing with the "default" browser is best.
+              Normally viewing the page in outside of the a web server context
```
Comment (glenfe) [inline development/website_updating.html:103]: "outside of the a web server context" should probably be "outside of a web server context"

Comment (libbooze) [reply development/website_updating.html]: Ah, wasn't looking at unmodified text. 
Fixed

**Pull request merged**